src/generate-hash.ts

```diff
@@ -0,0 +1,61 @@
+import { NonPrimitive } from '@superfaceai/one-sdk/dist/internal/interpreter/variables';
+import { createHash } from 'crypto';
+import createDebug from 'debug';
+
+const debugHashing = createDebug('superface:testing:hash');
+
+export interface IGenerator {
+  hash: (param: { input: NonPrimitive; testName?: string }) => string;
+}
+
+const generate = (value: string): string => {
+  debugHashing('Trying to generate hash based on specified value:', value);
+
+  return createHash('md5').update(value).digest('hex');
+};
+
+export class JestGenerateHash implements IGenerator {
```

Comment:
Implementation of `JestGenerateHash` and `MochaGenerateHash` are exactly the same except logging. I am wondering why do we have two implementations?

Reply:
When I was testing mocha's access to test names, it had different approach than jest. I was wondering having them separately would be good for possible extension of these components, but I can merge them until it's not useful.

The main difference is that jest object `expect` is not tied to test context, so wherever in test we instantiate `SuperfaceTest` with `expect` as test instance, it will have `getState()` function which will return object with current test name, whereas mocha is tied to tests, so we're not able to pass its instance wherever in test to get current test name and to enable that we might need to handle test instance at execution of method `run()`.

Reply:
OK. I understand that it is there for possible future extensions. Hopefully it will be useful.